src/main/java/com/study/stockmanagementstudycase/controller/WareHouseController.java

```diff
@@ -1,22 +1,29 @@
 package com.study.stockmanagementstudycase.controller;
 
 import com.study.stockmanagementstudycase.model.dto.request.WareHouseCreateRequest;
+import com.study.stockmanagementstudycase.model.dto.response.WareHouseResponse;
 import com.study.stockmanagementstudycase.model.entities.WareHouseEntity;
 import com.study.stockmanagementstudycase.service.WareHouseCreateService;
+import com.study.stockmanagementstudycase.service.WareHouseGetAllService;
 import jakarta.validation.Valid;
 import lombok.RequiredArgsConstructor;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
 @RequestMapping("api/v1/warehouses")
 @RequiredArgsConstructor
 public class WareHouseController {
 
     private final WareHouseCreateService wareHouseCreateService;
+    private final WareHouseGetAllService wareHouseGetAllService;
```

Comment:
WareHouse için get işlemlerini içeren servise WareHouseService şeklinde genel bir isimlendirme verebiliriz. Diğerlerinde ise Create, Update gibi kullanılmaya devam edilebilir.

Reply:
Tamamdir